### ISSUE TITLE
add react 18 support

### ISSUE THREAD

**Issue by cjquines** [nkbt/react-debounce-input#150]
think it works fine in react 18, just needs a peer dep bump

**Comment by nkbt** [comment]
Yeah right. Will do. Need to update dev/build deps too, should be ready soon
Thanks 🙏

**Comment by volter111** [comment]
any update on this?

**Comment by nkbt** [comment]
Oops, I did merge an update but totally forgot to publish 🤦🏻‍♂️ So much work

**Comment by Nithur-M** [comment]
Any update on this one?

**Comment by nkbt** [comment]
published
```
+ react-debounce-input@3.3.0
```



https://github.com/nkbt/react-debounce-input/releases/tag/v3.3.0
